people/qiqi233345/page.en.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Qi
+---
+Qi doesn't often talk on the Internet, but she is very enthusiastic in real life. When being with her, the funny she show always bring people happiness.
+She looks mysterious, and doesn't introduce herself very much because she thinks "I am me, you just know "me", you don't need to know "me with a certain label". If someone can't understand me, that's scolding me, don’t like me. I don't want he to hate people who have the same label or experience as me because of me.”
```

Comment:
looks -> seems
使用正确的英文双引号：“”

She seems mysterious. She didn't often introduce herself because she thinks: “I am me, I want you know me for who I am, not know me as someone with a certain label. If someone can't understand me, that's their problem. I don't want them to hate people who have the same label or experience as me because of me.”